README.md

```diff
@@ -38,119 +40,135 @@ The release can be found here: https://github.com/PDFTron/pdftron-react-native/r
 1. First, follow the official getting started guide on [setting up the React Native environment](https://reactnative.dev/docs/environment-setup), [setting up the iOS and Android environment](https://reactnative.dev/docs/environment-setup), and [creating a React Native project](https://reactnative.dev/docs/environment-setup). The following steps will assume your app is created through `react-native init MyApp`. This guide also applies if you are using the [TypeScript template](https://reactnative.dev/docs/environment-setup#optional-using-a-specific-version-or-template).
 
 2. In `MyApp` folder, install `react-native-pdftron` by calling:
-    ```shell
-    yarn add github:PDFTron/pdftron-react-native
-    yarn add @react-native-community/cli --dev
-    yarn add @react-native-community/cli-platform-android --dev
-    yarn add @react-native-community/cli-platform-ios --dev
-    yarn install
-    ```
-    or
-    ```shell
-    npm install github:PDFTron/pdftron-react-native --save
-    npm install @react-native-community/cli --save-dev
-    npm install @react-native-community/cli-platform-android --save-dev
-    npm install @react-native-community/cli-platform-ios --save-dev
-    npm install
-    ```
 
-### Android
+   ```shell
```

Comment:
i think it needs to be clearly stated with separate options, i.e 2a/2b, and please include full command, `react-native-pdftron` along is very confusing what it is

Reply:
changed